easybuild/tools/options.py

```diff
@@ -324,7 +325,7 @@ def informative_options(self):
                                          "easyblock-specific ones by using -e)"),
                                         'choice', 'store_or_None', FORMAT_TXT, [FORMAT_RST, FORMAT_TXT], 'a'),
             'avail-easyconfig-templates': (("Show all template names and template constants "
```

Comment:
this should use the global output format, so just boolean here? not store or none